mlog/log.go

```diff
@@ -161,20 +167,91 @@ func (l *Logger) Sugar() *SugarLogger {
 
 func (l *Logger) Debug(message string, fields ...Field) {
 	l.zap.Debug(message, fields...)
+	if l.logrLogger != nil && isLevelEnabled(l.logrLogger, logr.Debug) {
```

Comment:
Why are we still using zap? Is there going to be a future pr to move completely over to the new logging? Should it be one or the other, not both?

Reply:
Zap will be replaced and mlog simplified once it gets some mileage. The underlying Logr package has been used in at least one large closed source SaaS product, however that usage pattern is just one datapoint and I would like to see more usage before we replace Zap.